src/content/en/tools/chrome-devtools/index.md

```diff
@@ -19,94 +22,135 @@ Dogfood: Looking for the latest, [Chrome Canary](https://www.google.com/intl/en/
+<img src="images/panels/sources.png" alt="Sources Panel" class="attempt-right">
+Debug your JavaScript using breakpoints or connect your local files via Workspaces to use DevTools as a code editor.
+
+* [Get Started With Debugging JavaScript](/web/tools/chrome-devtools/javascript)
 * [Debugging with Breakpoints](/web/tools/chrome-devtools/javascript/add-breakpoints)
-* [Debug Obfuscated Code](/web/tools/chrome-devtools/javascript/add-breakpoints)
 * [Set Up Persistence with DevTools Workspaces](/web/tools/setup/setup-workflow)
+* [Run Snippets Of Code From Any Page](/web/tools/chrome-devtools/snippets)
 
 <div style="clear:both;"></div>
 
-### Network
-<img src="images/network-panel.png" alt="Network Panel" class="attempt-right">
-Use the Network panel to get insights into requested and downloaded resources and optimize your page load performance.
+### Network panel
+
+<img src="images/panels/network.png" alt="Network Panel" class="attempt-right">
 
-* [Network Panel Basics](/web/tools/chrome-devtools/network-performance/resource-loading)
-* [Understanding Resource Timing](/web/tools/chrome-devtools/network-performance/understanding-resource-timing)
-* [Network Throttling](/web/tools/chrome-devtools/network-performance/network-conditions)
+Optimize page load performance and debug request issues.
+
+* [Get Started](/web/tools/chrome-devtools/network-performance/)
+* [Network Issues Guide](/web/tools/chrome-devtools/network-performance/issues)
+* [Network Panel Reference](/web/tools/chrome-devtools/network-performance/reference)
 
 <div style="clear:both;"></div>
 
-### Timeline
-<img src="images/timeline-panel.png" alt="Timeline Panel" class="attempt-right">
-Use the Timeline to improve the run time performance of your page by recording and exploring the various events that happen during the lifecycle of a site.
+### Performance panel (previously Timeline panel)
+
+<img src="images/panels/performance.png" alt="Timeline Panel" class="attempt-right">
+Improve the runtime performance of your page by recording and exploring the various events that happen during the lifecycle of a site.
+
+<b>Note</b>: In Chrome 57 the Timeline panel was renamed to the Performance panel.
 
 * [How to look at performance](/web/tools/chrome-devtools/evaluate-performance/timeline-tool)
 * [Analyze runtime performance](/web/tools/chrome-devtools/rendering-tools/)
 * [Diagnose Forced Synchronous Layouts](/web/tools/chrome-devtools/rendering-tools/forced-synchronous-layouts)
 
 <div style="clear:both;"></div>
 
-### Profiles
-<img src="images/profiles-panel.png" alt="Profiles Panel" class="attempt-right">
-Use the Profiles panel if you need more information than the Timeline provide, for instance to track down memory leaks.
+### Memory panel (previously Profiles panel)
+
+<img src="images/panels/memory.png" alt="Profiles Panel" class="attempt-right">
+Profile memory usage and track down leaks.
 
+<b>Note</b>: In Chrome 57 the Profiles panel was renamed to the Memory panel.
```

Comment:
Please use the standard `Note: something` syntax so you get the DevSite styled note stuff.